src/MarginTrading.Backend.Core/OvernightSwapCalculation.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace MarginTrading.Backend.Core
+{
+	public class OvernightSwapCalculation : IOvernightSwapHistory, IOvernightSwapState
```

Comment:
Why need both if IOvernightSwapHistory : IOvernightSwapState ?

Reply:
To make code more readable.